library/Core.cpp

```diff
@@ -2266,6 +2266,7 @@ bool Core::ncurses_wgetch(int in, int & out)
     }
     if(in >= KEY_F(1) && in <= KEY_F(8))
     {
+/* TODO: understand how this changes for v50
```

Comment:
What part of this is problematic? As far as I can tell, hotkeys are about the same.

Reply:
the reference to `ui`. It's just a name change, but the new names aren't actually in the tree yet.

Reply:
and `ui_sidebar_mode`